src/utils/cacheRequest.js

```diff
@@ -63,38 +64,34 @@ const fetchCachedGeneExpressionWork = async (experimentId, timeout, body, backen
   return responseData;
 };
 
-const fetchCachedWork = async (experimentId, timeout, body, backendStatus) => {
+const fetchCachedWork = async (
+  experimentId,
+  timeout,
+  body,
+  qcPipelineStartDate,
```

Comment:
Any reason for not sending the entire body over as we did before? It is probably a better thing to do in case we ever need some other checks here.

Reply:
`body` is sent as before, there is no change here. The change is in the additional `additionaSendWorkParams` parameter, as a way to pass additional configuration to `sendWork`

Reply:
fixed to include the entire `backendStatus.status` body into `fetchCachedWork`